django_restql/mixins.py

```diff
@@ -4,6 +4,7 @@
     Serializer, ListSerializer,
     ValidationError
 )
+from django.db.models import Prefetch
```

Comment:
Can we consider adding flake8 to the CI run? It can help fail the build if we want this to be a blocker that you don't want to scour for.

Reply:
@yezyilomo We actually do use it for type checking on the apply function.

Reply:
> Can we consider adding flake8 to the CI run? It can help fail the build if we want this to be a blocker that you don't want to scour for.

I think for now it's fine..